File: C++/Estrutura sequencial/consumo.cpp
// Biblioteca
#include <bits/stdc++.h>

using namespace std;

int main()
{
    // Variáveis
    int distanciaPercorrida;
    double combustivelGasto, consumoMedio;

    // Entrada de dados
    cout << "Distancia percorrida: ";
    cin >> distanciaPercorrida;

    cout << "Combustivel gasto: ";
    cin >> combustivelGasto;

    // Algoritmo matemático
    consumoMedio = (double) distanciaPercorrida / combustivelGasto;

    // Saída de dados
    cout << fixed << setprecision(3); // Quantidade de casas decimais
    cout << "O consumo medio e de " << consumoMedio;

    return 0;
}

File: Java/Estrutura sequencial/consumo.java
// Importando biblioteca
import java.util.Scanner;

public class consumo {

	public static void main(String[] args) {
		
		// Variáveis
		int distanciaPercorrida;
		double combustivelGasto, consumoMedio;

		Scanner sc = new Scanner(System.in); // Para facilitar a leitura de dados
		
		// Entrada de dados
		System.out.print("Distância percorrida: ");
		distanciaPercorrida = sc.nextInt();
		System.out.print("Combustível gasto: ");
		combustivelGasto = sc.nextDouble();
		
		// Algoritmo matemático
		consumoMedio = distanciaPercorrida / combustivelGasto;
		
		// Saída de dados
		System.out.print("O consumo médio é de " + String.format("%.3f", consumoMedio) +  ".");
		
		sc.close(); // Boa prática de programação
	}
}